js/base/Exchange.js

```diff
@@ -797,6 +797,15 @@ module.exports = class Exchange {
     // ------------------------------------------------------------------------
     // METHODS BELOW THIS LINE ARE TRANSPILED FROM JAVASCRIPT TO PYTHON AND PHP
 
+    getDefaultOptions () {
+        return {
+            'defaultNetworkCodeReplacements': {
```

Comment:
@ttodua, to avoid all of the extra logic and added methods since this is very stationary information, what do you think about keeping this dictionary inside `networkIdToCode` instead?

Reply:
we make overrides of `networkIdToCode` in derived classes, so we end up having duplicated codes, which is more undesirable, than adding some code in describe()/options. we should have that `defaultNetworkCodeReplacements` dictionary only on one place, I believe so.

Reply:
@ttodua fair enough 🙂